README.md

```diff
@@ -18,19 +18,232 @@ Alternatively, you can use the Open in Colab browser extension ([Chrome](https:/
+        integer id PK 
+        text project_id 
+    }
+
+    run_collection {
+        integer collection_id FK,UK 
+        integer id PK 
+        character_varying run_id UK 
+    }
+
+    check_results }o--|| collection : "collection_id"
+    collection_check }o--|| collection : "collection_id"
+    field_counts }o--|| collection : "collection_id"
+    indicator_coverage }o--|| collection : "collection_id"
+    project_fields }o--|| collection : "collection_id"
+    projects }o--|| collection : "collection_id"
+    run_collection }o--|| collection : "collection_id"
+
+```
+
+### Tables
+
+The following table lists all tables in the database. For information on the columns in each table, refer to the subheadings below the table.
+
+| Table                | Description                                                                                                                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [collection](#collection)          | A collection is an OC4IDS dataset.                                                                                                                                                     |
+| [collection_check](#collection_check)    | libcoveoc4ids CLI output for each collection, i.e. the results reported by the OC4IDS Data Review Tool. For more information, see https://github.com/open-contracting/lib-cove-oc4ids/ |
+| [projects](#projects)            | Projects in OC4IDS JSON format.                                                                                                                                                        |
+| [field_counts](#field_counts)        | Counts of the occurrences of fields in each collection.                                                                                                                                  |
+| [project_fields](#project_fields)      | The fields present in each project.                                                                                                                                                    |
+| [check_results](#check_results)       | Results of running checks on data quality. For more information, see the quality criteria, checks and metrics notebook in https://github.com/open-contracting/notebooks-oc4ids         |
+| [run_collection](#run_collection)      | Relationships between collections and check runs.                                                                                                                                      |
+| [indicator_coverage](#indicator_coverage)  |                                                                                                                                                                                        |
```

Comment:
no description?

Reply:
Ah, good spot. Added in https://github.com/open-contracting/notebooks-oc4ids/pull/48/commits/68ece08c503aa45303663e693f0ad24917e01b14